seacrowd/sea_datasets/bactrian_x/bactrian_x.py

```diff
@@ -0,0 +1,153 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Tasks, Licenses, TASK_TO_SCHEMA, SCHEMA_TO_FEATURES
+
+_CITATION = """\
+@misc{li2023bactrianx,
+      title={Bactrian-X : A Multilingual Replicable Instruction-Following Model with Low-Rank Adaptation}, 
+      author={Haonan Li and Fajri Koto and Minghao Wu and Alham Fikri Aji and Timothy Baldwin},
+      year={2023},
+      eprint={2305.15011},
+      archivePrefix={arXiv},
+      primaryClass={cs.CL}
+}
```

Comment:
Unsure if we should follow this and cite the other papers (seem unrelated to the dataset)

![image](https://github.com/SEACrowd/seacrowd-datahub/assets/12949683/6f63e414-6cac-48dc-a71e-461821212e45)